test/e2e/functional_test.go

```diff
@@ -28,7 +28,7 @@ func (s *FunctionalSuite) TestArchiveStrategies() {
 		Workflow(`@testdata/archive-strategies.yaml`).
 		When().
 		SubmitWorkflow().
-		WaitForWorkflow(time.Minute).
+		WaitForWorkflow(time.Second * 90).
```

Comment:
How did you go about finding all the places to change this? If I do "Find all References" from my VSCODE IDE for `WaitForWorkflow()`, I think I see a bunch of other places where a time is being passed in (agent_test.go, cron_test.go, etc)

Reply:
(if you change them I'd say you don't need to strictly add 30 seconds each time by the way - adding a little extra is good too - e.g. I think it would be weird to change 5 to 35 and might as well round up to 45 seconds or a minute)

Reply:
If I recall correctly, I only changed the instances where the tests were failing.

Reply:
Have you tried to do it any time a test failed even once? I’m a little concerned about creating new failures that are very intermittent but sometimes do time out

Reply:
Yeah, I haven't seen failures for the other cases. I've been keeping an eye on that as a potential issue.

If you want to ensure the exact behavior as before, I can add 30 seconds to all cases. I can round up to the nearest 15 seconds as you've suggested above, too.

Reply:
That would be really great! Thank you!